code/datums/statistics/entities/death_stats.dm

```diff
@@ -136,9 +136,9 @@
 
 /mob/living/carbon/Xenomorph/track_mob_death(var/cause, var/cause_mob)
 	var/datum/entity/statistic/death/new_death = ..(cause, cause_mob, caste_type)
+	new_death.is_xeno = TRUE // this was placed beneath the if below, which meant gibbing as a xeno wouldn't track properly in stats
```

Comment:
add an if check for whether new_death exists

Reply:
check added